datadog_lambda/tracing.py

```diff
@@ -801,6 +808,7 @@ def create_inferred_span_from_api_gateway_websocket_event(
         "connection_id": request_context.get("connectionId"),
         "event_type": request_context.get("eventType"),
         "message_direction": request_context.get("messageDirection"),
+        "peer.service": service_env_var,
```

Comment:
💡  just a general comment and it doesn't have to be done in this pr: we should create a `general_tags()` method and save some repetition.

Reply:
I agree, we should refactor this, but it's a really long file. We should follow a similar approach in Node.